client/index.html

```diff
@@ -7,6 +7,14 @@
 </head>
 
 <body>
+  
+
+  <ui-view></ui-view>
+
```

Comment:
Not test wise. Was receiving weird errors, so decided to move some things around. Everything still works. Added white space mainly for quick visual as to where things started and ended. Can edit if needed.